src/components/collapsible_nav_beta/collapsible_nav_item/collapsible_nav_item.tsx

```diff
@@ -65,11 +60,11 @@ export type _SharedEuiCollapsibleNavItemProps = HTMLAttributes<HTMLElement> &
 
 export type EuiCollapsibleNavItemProps = {
   /**
-   * ReactNode to render as this component's title
+   * Required Text to render as the nav item title
```

Comment:
whoops, just noticed this typo
```suggestion
   * Required text to render as the nav item title
```